### DISCUSSION TITLE
Issue Deploying Hoop.dev with ArgoCD: 403 Forbidden on Helm Chart Repository

### DISCUSSION THREAD

**Discussion by femiadewale780** [hoophq/hoop#523] [Q&A] [answered]
Hello
I'm trying to deploy Hoop.dev on EKS using ArgoCD. My configuration seems correct, but I’m running into an issue when pulling the Helm chart. Here is my ArgoCD Application manifest:

```
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: hoop-dev
  namespace: argocd
  finalizers:
    - resources-finalizer.argocd.argoproj.io
spec:
  project: cluster
  syncPolicy:
    automated:
      prune: true
      selfHeal: true
    syncOptions:
      - CreateNamespace=true
  source:
    chart: hoop-chart
    repoURL: https://releases.hoop.dev
    targetRevision: "1.27.6"
    helm:
      valueFiles:
        - values.yaml
  destination:
    name: dev
    namespace: hoop-dev
```

When trying to sync the application, I’m getting the following error:
```
ComparisonError
Failed to load target state: failed to generate manifest for source 1 of 1: rpc error: code = Unknown desc = `helm pull --destination /tmp/d478c612-af08-4143-9982-f46b18551723 --version 1.27.6 --repo https://releases.hoop.dev/ hoop-chart` failed exit status 1: 
Error: looks like "https://releases.hoop.dev/" is not a valid chart repository or cannot be reached: failed to fetch https://releases.hoop.dev/index.yaml : 403 Forbidden
```
i have tested locally, tried other urls and chart name but got the same 403 error.

Any help to resolve this would be appreciated!

Doc used https://hoop.dev/docs/getting-started/installation/kubernetes

**Comment by sandromello** [comment, marked as answer]
Hello there!
The url `https://releases.hoop.dev` is not a helm chart repository. You need to host your own repository or use the helm template to generate the assets based on a specific version.
